spk/nzbget/src/nzbget.conf

```diff
@@ -85,8 +85,14 @@ ScriptDir=/usr/local/nzbget/share/nzbget/scripts
 
```

Comment:
I don't remember why it's 755 but I believe it should be 644.
That doesn't change anything anyway because the file is installed with 644: https://github.com/SynoCommunity/spksrc/blob/master/spk/nzbget/Makefile#L39